ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: openshift-pipelines-operator-rh
   namespace: openshift-operators
 spec:
-  channel: pipelines-1.10
+  channel: pipelines-1.11
```

Comment:
Do we always want to install from the latest channel ?
Shouldn't we add the channel as an optional variable for the test (and from Jenkins parameters as well) ?